<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WinSaveImg</class>
 <widget class="QWidget" name="WinSaveImg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>771</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="labelCurrentImg">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>40</y>
     <width>401</width>
     <height>521</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>540</y>
     <width>311</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="cancelButton">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="saveButton">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>Save image</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>291</width>
     <height>241</height>
    </rect>
   </property>
   <property name="title">
    <string>Depth color</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>271</width>
      <height>201</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="depth256Button">
       <property name="text">
        <string>256 - Color</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="depth16Button">
       <property name="text">
        <string>16 - Hight Color</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="depth24Button">
       <property name="text">
        <string>24 - True Color</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="depth32Button">
       <property name="text">
        <string>32 - Color</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
